src/modules/ui/components/left_panel/stories.js

```diff
@@ -49,49 +52,49 @@ class Stories extends React.Component {
     }
 
     return (
-      <div {...props}>
-        {story}
-      </div>
+      <li key={story}>
+        <a style={style} onClick={props.onClick}>{story}</a>
+      </li>
     );
   }
 
   renderKind({ kind, stories }) {
     const { selectedKind } = this.props;
-    const style = { ...kindStyle };
+    const style = { display: 'block', ...kindStyle };
+    const onClick = this.fireOnKind.bind(this, kind);
 
     if (kind === selectedKind) {
       style.fontWeight = 'bold';
       return (
-        <div key={kind}>
-          <div
-            style={style}
-            onClick={this.fireOnKind.bind(this, kind)}
-          >
+        <li key={kind}>
+          <a style={style} onClick={onClick}>
```

Comment:
Opened a PR but we need to verify whether improve accessibility before merging
https://github.com/kadirahq/storybook-ui/pull/44